src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunItems.java

```diff
@@ -750,6 +750,7 @@ private SlimefunItems() {}
     public static final SlimefunItemStack BLISTERING_INGOT_3 = new SlimefunItemStack("BLISTERING_INGOT_3", Material.GOLD_INGOT, "&6Blistering Ingot", "", LoreBuilder.radioactive(Radioactivity.VERY_HIGH), LoreBuilder.HAZMAT_SUIT_REQUIRED);
 
     public static final SlimefunItemStack ENERGY_REGULATOR = new SlimefunItemStack("ENERGY_REGULATOR", HeadTexture.ENERGY_REGULATOR, "&6Energy Regulator", "", "&fCore Component of an Energy Network");
+    public static final SlimefunItemStack ENERGY_CONNECTOR = new SlimefunItemStack("ENERGY_CONNECTOR", HeadTexture.CARGO_CONNECTOR_NODE, "&cEnergy Connector", "", "&fEnergy Connector Wire");
```

Comment:
Only thing missing is changing the texture here, I think you created one and posted it somewhere?
Just figured I'd review the changes and bump this since you requested a review a couple of days ago

Reply:
> Only thing missing is changing the texture here, I think you created one and posted it somewhere?
> Just figured I'd review the changes and bump this since you requested a review a couple of days ago

Yeah the texture is ready. I'll add it to the code and to mojand database soon.